packages/devtools_app/lib/src/app.dart

```diff
@@ -48,6 +48,18 @@ import 'utils.dart';
 import 'vm_developer/vm_developer_tools_controller.dart';
 import 'vm_developer/vm_developer_tools_screen.dart';
 
+// Assign to true to use a sample implementation of a conditional screen.
+// WARNING: Do not check in this file with enableExample set to true.
+const enableSampleScreen = false;
```

Comment:
nit: call this debugEnableSampleScreen to match the flutter convention for top level fields that are debug toggles.

Reply:
Done